Source/CoreClasses/Game.swift

```diff
@@ -0,0 +1,59 @@
+//
+//  Game.swift
+//  CommonFiles
+//
+//  Created by Tiewhan Smith on 2020/02/23.
+//
+
+import Foundation
+
+///Represents a digital game.
+public class Game {
+
+  public var appID: String
+  public var name: String
+  private var corePrice: Double?
+  
+  // MARK: All properties (Except appID and name) need to be set to null by the reduce memory load
+  // MARK: method
+  var price: Double {
+  
+    get {
+  
```

Comment:
Don't you think the spacing in this file is just a liiiittle bit much. Please could you take off a few trailing spaces?